mdbenchmark/generate.py

```diff
@@ -119,22 +89,24 @@ def generate(name, gpu, module, host, min_nodes, max_nodes, time, list_hosts):
             'Could not find template for host \'{}\'.'.format(host),
             param_hint='"--host"')
 
-    if not module:
-        raise click.BadParameter(
-            'You did not specify which gromacs module to use for scaling.',
-            param_hint='"-m" / "--module"')
-
-    # Provide some output for the user
-    number_of_mdbenchmarks = len(module) * max_nodes
-    run_time_each = '{} minutes'.format(time)
-    console.info(
-        'Creating a total of {} benchmarks, with a run time of {} each.',
-        number_of_mdbenchmarks, run_time_each)
-
     for m in module:
         directory = '{}_{}'.format(host, m)
         gpu_string = '.'
 
+        # Here we detect the mdengine (GROMACS or NAMD).
+        # Note: NAMD input files can have relative imports. To avoid issues
+        # when copying the files, we currently only allow the use of absolute
+        # paths.
+        engine = detect_md_engine(m)
+        if engine == namd:
```

Comment:
Could you please put this into one single `console.warn` call? Also we probably can skip the first line and only use something like this: `console.warn('NAMD support is experimental. All input files must be in the current directory. Parameter paths must be absolute. Only crude file checks are performed!')`